src/app/server/CommissioningWindowManager.cpp

```diff
@@ -89,8 +89,8 @@ void CommissioningWindowManager::OnPlatformEvent(const DeviceLayer::ChipDeviceEv
 #if CONFIG_NETWORK_LAYER_BLE
     else if (event->Type == DeviceLayer::DeviceEventType::kCloseAllBleConnections)
     {
-        ChipLogProgress(AppServer, "Received kCloseAllBleConnections");
-        mServer->GetBleLayerObject()->CloseAllBleConnections();
+        ChipLogProgress(AppServer, "Received kCloseAllBleConnections:%d", event->Type);
+        mServer->GetBleLayerObject()->Shutdown();
```

Comment:
Shutdown != to CloseAllBleConnections. How widely was this change tested? It impacts all BLE users.

Reply:
As we are in Non-concurrent mode, the BLE must be stopped for all BLE users. The BLE will be either powered off or just the use of the RF disabled. as we are commissioning I am not aware of any other BLE users at this time.